.env.sample

```diff
@@ -1,7 +0,0 @@
-REACT_APP_MB_ACCESS_TOKEN=""
-ALGOLIA_APP_ID=""
-ALGOLIA_API_KEY=""
-NODE_ENV="development"
-FIREBASE_EMULATOR_HOST_VAR=localhost:9000
-FIRESTORE_EMULATOR_HOST=localhost:8080
-GCLOUD_PROJECT=mutualaid-757f6
```

Comment:
Hey @marekdano may have botched one of your conflict resolutions.

How come we're removing this file?

Reply:
The file was generated and I thought I wouldn't need it when I copied the content to `.env` file. I'm reversing the change to have the file in the codebase.